tests/manage/monitoring/prometheus/test_capacity.py

```diff
@@ -44,17 +45,36 @@ def test_capacity_workload_alerts(
     alerts = workload_storageutilization_95p.get('prometheus_alerts')
     # TODO(fbalak): it seems that CephFS utilization triggers only firing
     # alerts. This needs to be more investigated.
+
+    if config.ENV_DATA.get('ocs_version') == '4.2':
+        nearfull_message = (
+            'Storage cluster is nearing full. Expansion is required.'
+        )
+        criticallfull_mesage = (
+            'Storage cluster is critically full and needs immediate expansion'
+        )
+    else:
+        # since OCS 4.3
+        nearfull_message = (
+            'Storage cluster is nearing full. Data deletion or cluster '
+            'expansion is required.'
+        )
+        criticallfull_mesage = (
+            'Storage cluster is critically full and needs immediate data '
+            'deletion or cluster expansion.'
+        )
+
     for target_label, target_msg, target_states, target_severity in [
         (
             constants.ALERT_CLUSTERNEARFULL,
-            'Storage cluster is nearing full. Expansion is required.',
-            ['pending', 'firing'] if interface == 'rbd' else ['firing'],
+            nearfull_message,
+            ['pending', 'firing'],
             'warning'
         ),
         (
             constants.ALERT_CLUSTERCRITICALLYFULL,
-            'Storage cluster is critically full and needs immediate expansion',
-            ['pending', 'firing'] if interface == 'rbd' else ['firing'],
+            criticallfull_mesage,
```

Comment:
This looks like removal of hidden workaround ... but I'm assuming that since you are removing it now, it's no longer a problem. In the future, it would be better to track this as a workaround from the start.